.gitignore

```diff
@@ -95,3 +95,4 @@ CorsixTH/CorsixTH.sln
 
 # Default build path
 CorsixTH/CorsixTH
+/.metadata/
```

Comment:
What is `.metadata`?

Reply:
Eclipse projekt data
you can ignore this